tests/jobserver/views/test_projects.py

```diff
@@ -54,6 +54,7 @@ def test_projectacceptinvite_success(rf):
     invite.refresh_from_db()
     assert invite.membership is not None
     assert invite.membership.project == project
+    assert invite.membership.roles == invite.roles
```

Comment:
We never assert anything about what `invite.roles` is.  Is it definitely set to a non-empty list?